src/AdapterPluginManager.php

```diff
@@ -39,14 +40,53 @@ class AdapterPluginManager extends AbstractPluginManager
     ];
 
     protected $factories = [
-        Adapter\IgBinary::class     => InvokableFactory::class,
-        Adapter\Json::class         => InvokableFactory::class,
-        Adapter\MsgPack::class      => InvokableFactory::class,
-        Adapter\PhpCode::class      => InvokableFactory::class,
-        Adapter\PhpSerialize::class => InvokableFactory::class,
-        Adapter\PythonPickle::class => InvokableFactory::class,
-        Adapter\Wddx::class         => InvokableFactory::class
+        Adapter\IgBinary::class             => InvokableFactory::class,
+        'zendserializeradapterigbinary'     => InvokableFactory::class,
+        Adapter\Json::class                 => InvokableFactory::class,
+        'zendserializeradapterjson'         => InvokableFactory::class,
+        Adapter\MsgPack::class              => InvokableFactory::class,
+        'zendserializeradaptermsgpack'      => InvokableFactory::class,
+        Adapter\PhpCode::class              => InvokableFactory::class,
+        'zendserializeradapter\phpcode'     => InvokableFactory::class,
+        Adapter\PhpSerialize::class         => InvokableFactory::class,
+        'zendserializeradapterphpserialize' => InvokableFactory::class,
+        Adapter\PythonPickle::class         => InvokableFactory::class,
+        'zendserializeradapterpythonpickle' => InvokableFactory::class,
+        Adapter\Wddx::class                 => InvokableFactory::class,
+        'zendserializeradapterwddx'         => InvokableFactory::class
```

Comment:
I've opened #11 to demonstrate. It looks like @ezimuel added a number of aliases when he merged your changes; I've added a few more. I find that we typically need each of:

- lowercase
- camelCase
- TitleCase

to hit all the various permutations used be folks when fetching from the plugin managers.